Client/mods/deathmatch/logic/CIFPAnimations.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __CIFPANIMATIONS_H
```

Comment:
File is missing the copyright/etc. comment at the top.
Also, use **#pragma once**